spread.yaml

```diff
@@ -581,7 +593,7 @@ prepare: |
         esac
         rm -f "$tf"
         curl -sS -o - "https://codeload.github.com/snapcore/snapd/tar.gz/$DELTA_REF" | gunzip > delta-ref.tar
-        xdelta3 -q -d -s delta-ref.tar current.delta | tar x --strip-components=1
+        xdelta3 -q -c -d -s delta-ref.tar current.delta | tar x --strip-components=1
```

Comment:
`-c` means _use stdout_ but we always used stdout, is this just for completeness or has something more complex happened here?

Reply:
Actually more happend. Before the pack would use `xdelta3 -S none -s delta-ref.tar <&3 > current.delta`, so the name of the new file was not known since the data was read from a file descriptor. Right now, there is a name, so when extracking, xdelta by default will take the input, apply the delta and create the output file named like the one that was used as the 'new'. Instead we need to have xdelta to use stdout to keep things simple.